backend/src/main/java/com/htecgroup/skynest/security/WebSecurity.java

```diff
@@ -42,6 +42,8 @@ protected void configure(HttpSecurity http) throws Exception {
         .authorizeRequests()
         .antMatchers(String.format("%s/**", UrlUtil.PUBLIC_CONTROLLER_URL))
         .permitAll()
+        .antMatchers("/token/refresh")
+        .permitAll()
```

Comment:
or you could change the authorization filter to use the RefreshToken header when servletPath is /token/refresh

this makes it seem like the client isn't authorized at all, when the client is just using a refresh token to authorize

Reply:
no no.. don't do that :) the main reason why refresh token is used is to create new access token, that is... don't change the flow :) btw I already explained here how should work: https://github.com/JDP2022SkyNest/SkyNest/pull/67#issuecomment-1162275844

Reply:
The problem is if the access token is expired and the endpoint is public I can't take the current logged user info and if I don't have that, I can't make new access token.

Reply:
but the refresh token needs to have the user id right? so you can extract that from there